static/app/views/replays/detail/network/index.tsx

```diff
@@ -113,6 +121,36 @@ function NetworkList({networkSpans, startTimestampMs}: Props) {
   return (
     <FluidHeight>
       <NetworkFilters networkSpans={networkSpans} {...filterProps} />
+      <Feature
+        features={['session-replay-network-details']}
+        organization={organization}
+        renderDisabled={false}
+      >
+        {isDismissed ? null : (
+          <StyledAlert
+            icon={<IconInfo />}
+            opaque={false}
+            showIcon
+            type="info"
+            trailingItems={
+              <StyledButton priority="link" size="sm" onClick={() => {}}>
+                <IconClose color="gray500" size="sm" />
+              </StyledButton>
+            }
+          >
+            {tct('You can now see the body of requests and responses. [link]', {
```

Comment:
I think this kind of implies we have been capturing this data and they can now see it vs this feature is now available, here's what you need to do to use it

```suggestion
            {tct('Start collecting the body of requests and responses. [link]', {
```